### ISSUE TITLE
Do you have a plan to release the training code to reproduce the paper results?

### ISSUE THREAD

**Issue by TatsuyaShirakawa** [zma-c-137/VarGFaceNet#4]
Do you have a plan to release the training code to reproduce the paper results?
Thanks.

**Comment by yanmengjia** [comment]
Hi,
We will give you details of training if you update your questions in issues. Thanks.

**Comment by clhne** [comment]
@TatsuyaShirakawa @yanmengjia 
Hi, all
Could you give the details of training, include the hyper-parameter settings, etc.
Look forward to your reply.
Thanks.

**Comment by Coderx7** [comment]
I'm looking for this as well. 
Could you kindly provide the codes and training scripts so the results can be replicated and we hopefully can better experiment with the code . 
Its greatly appreciated

**Comment by nullptr-leo** [comment]
@yanmengjia 
请问下 recursive=1 的蒸馏就是普通蒸馏吗？还是说普通蒸馏完再来一次？
或者换句话说，有没有 recursive=0 这种配置？

**Comment by anandhakrishnanh** [comment]
@zma-c-137  Looking forward for the training code as well. The accuracy you claim seems very impressive.
